.ci/openvino-onnx/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM ubuntu:20.04
+
+LABEL version=2020.07.03.2
+
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy ${http_proxy}
+ENV https_proxy ${https_proxy}
+
+ENV CI=true
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PYTHONUNBUFFERED 1
+
+# Install base dependencies
+RUN apt-get update && apt-get install -y locales && apt-get clean autoclean && apt-get autoremove -y
+
+# Set the locale to en_US.UTF-8
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# OpenVINO dependencies
+RUN apt-get update && apt-get -y --no-install-recommends install \
+        autoconf \
+        automake \
+        build-essential \
+        cmake \
+        curl \
+        git \
+        libtool \
+        ocl-icd-opencl-dev \
+        pkg-config \
+        unzip \
+        wget && \
+  apt-get clean autoclean && \
+  apt-get autoremove -y
+
+# Python dependencies
+RUN apt-get -y --no-install-recommends install \
+        python3 \
+        python3-pip \
+        python3-dev \
+        python3-virtualenv \
+        cython3 \
+        tox && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# ONNX dependencies
+RUN apt-get -y --no-install-recommends install \
+        git-lfs \
+        protobuf-compiler \
+        libprotobuf-dev && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# Build OpenVINO
+COPY . /openvino/
+WORKDIR /openvino/build
+RUN cmake .. \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -DENABLE_VALIDATION_SET=OFF \
+    -DENABLE_VPU=OFF \
+    -DENABLE_DLIA=OFF \
+    -DENABLE_GNA=OFF \
+    -DENABLE_OPENCV=OFF \
+    -DENABLE_CPPLINT=OFF \
+    -DENABLE_TESTS=OFF \
+    -DENABLE_BEH_TESTS=OFF \
+    -DENABLE_FUNCTIONAL_TESTS=OFF \
+    -DENABLE_MKL_DNN=ON \
+    -DENABLE_CLDNN=OFF \
+    -DENABLE_PROFILING_ITT=OFF \
+    -DENABLE_SAMPLES=OFF \
+    -DENABLE_SPEECH_DEMO=OFF \
+    -DENABLE_PYTHON=ON \
+    -DPYTHON_EXECUTABLE=/usr/bin/python3 \
+    -DNGRAPH_ONNX_IMPORT_ENABLE=ON \
+    -DNGRAPH_IE_ENABLE=ON \
+    -DNGRAPH_INTERPRETER_ENABLE=ON \
+    -DNGRAPH_DEBUG_ENABLE=OFF \
+    -DNGRAPH_DYNAMIC_COMPONENTS_ENABLE=ON \
+    -DCMAKE_INSTALL_PREFIX=/openvino/dist
+RUN make -j 88 install
```

Comment:
More fancy... :)
```suggestion
RUN make -j `lscpu --parse=CORE | grep -v '#' | sort | uniq | wc -l` install
```

Reply:
Why not just -j$(nproc)?

Reply:
@nkogteva  The above one liner make sure to use all physical cores and not all logical cores as it does `nproc`.

Reply:
Changed to `-j$(nproc)`